krod/interfaces/commands/vector_store.py

```
@@ -0,0 +1,56 @@
+import click
+from typing import Optional
```

Ruff (0.11.9)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)